examples/data-objects/todo/src/index.ts

```diff
@@ -89,12 +92,28 @@ class TodoContainerRuntimeFactory extends BaseContainerRuntimeFactory {
 			registryEntries: new Map([TodoFactory.registryEntry]),
 			// eslint-disable-next-line import/no-deprecated
 			requestHandlers: [mountableViewRequestHandler(MountableView, [todoRequestHandler])],
-			provideEntryPoint: async (containerRuntime: IContainerRuntime) => {
-				const entryPoint = await containerRuntime.getAliasedDataStoreEntryPoint(todoId);
-				if (entryPoint === undefined) {
-					throw new Error("default dataStore must exist");
+			provideEntryPoint: async (
+				runtime: IContainerRuntime,
+			): Promise<IFluidMountableViewEntryPoint> => {
+				const todoDataObject = await getDataStoreEntryPoint<Todo>(runtime, todoId);
+
+				// TODO: need to incorporate logic from todoRequestHandler
```

Comment:
@ChumpChief Any quick idea on how to handle returning either `TodoView` or `TodoItemView` for the entryPoint? It's dependent on the request url, but I don't see any request calls in this example (all seem to be in webpack-fluid-loader).

Reply:
Ah, annoying :)

In a well-built production scenario, the host would own the URL format and parse it to provide the user the appropriate view/data.  However, since WFL is built to be a generic host it needs to delegate any demo-specific logic to those respective demos.  Which means the Todo demo owns path parsing, unfortunately.

So I see maybe two options:
1. Instead of the WFL entrypoint contract being `getDefaultDataObject()`, `getDefaultView()`, `getMountableDefaultView()` they become `getDataObject(path?)`, `getView(path?)`, `getMountableView(path?)` - representing that this host (WFL) specifically delegates path parsing to the container as part of its contract.  Most demos would do nothing with this and return the default object when the path is omitted.  Downside is that this is still very close to the request pattern, but it does formalize that relationship accurately between WFL and demo.
    * Now that I think of it, maybe you don't need `getDataObject(path?)` - does the container ever reach to its own entrypoint or does it always use internal methods to retrieve data objects?  WFL probably never requests this at least.  Removing this would align with the WFL demos being "container-view"-style bundlings that don't intend to expose their data objects outside the container.
3. Have this demo reach to the `location.pathname` and make some secret assumptions about the format.  Definitely more of a hack though, so probably prefer the first one?

Reply:
If you go with option 1, you'd only want to pass the non-WFL portion of the path -- e.g. if the url is `http://localhost:8080/doc/791f498c-1eea-452a-8b5c-79a942fad232/foobar` only passing `foobar`